Gaijinizer patch/Map537.txt

```diff
@@ -4,243 +4,310 @@
+But if you really want to fight me no matter
+what, I suppose I'll take you on.#Ron
 > END STRING
 
 > BEGIN STRING
 ？？？「対象、発見」
-> CONTEXT: Map537/events/7/pages/0/150/Dialogue < UNTRANSLATED
-
+> CONTEXT: Map537/events/7/pages/0/150/Dialogue
+??? 「Target... confirmed...」#Ron
 > END STRING
 
 > BEGIN STRING
 第零種断界接触、および第二法則侵蝕率0.986……
 イレギュラー個体を処断する。
-> CONTEXT: Map537/events/7/pages/0/161/Dialogue/kureo_adora_fc1 < UNTRANSLATED
+> CONTEXT: Map537/events/7/pages/0/161/Dialogue/kureo_adora_fc1
+Class Zero Cross-World Contact detected...
+Second Law Corruption Rate of 0.986%...
 
+Passing sentence on irregularity...#Ron
 > END STRING
 
 > BEGIN STRING
 わーお………
 
 まさかの………
-> CONTEXT: Map537/events/7/pages/0/164/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: Map537/events/7/pages/0/164/Dialogue/enemy29
+Waah...
 
+Is that...#Ron
 > END STRING
 
 > BEGIN STRING
 キミは、受肉した規定法則……
 
 ……たぶん第四法則あたりかな。
-> CONTEXT: Map537/events/7/pages/0/182/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: Map537/events/7/pages/0/182/Dialogue/enemy29
+You're an incarnation of natural law...
 
+You probably correspond to the fourth one...#Ron
 > END STRING
 
 > BEGIN STRING
 いやー助かったよ！
 
 ボクとしても、これ以上いざこざに巻き込まれたくなかったし
 キミが来てくれなかったらどうなってたことか……
-> CONTEXT: Map537/events/7/pages/0/186/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map537/events/7/pages/0/186/Dialogue/enemy29
+Haah, you really did me a solid!
+I don't know what I would've done if you hadn't
+come, I didn't want to get wrapped up in any more
+trouble...#Ron
 > END STRING
 
 > BEGIN STRING
 我々は、もはや法則でも概念でもない。
 
 この意志をもって、世界を混沌に沈める……
-> CONTEXT: Map537/events/7/pages/0/191/Dialogue/kureo_adora_fc1 < UNTRANSLATED
+> CONTEXT: Map537/events/7/pages/0/191/Dialogue/kureo_adora_fc1
+We are no longer natural laws or concepts...
 
+We will use our will to plunge the world into
+chaos.#Ron
 > END STRING
 
 > BEGIN STRING
 …………………！！
 
 …あ、そういうヤツ？
-> CONTEXT: Map537/events/7/pages/0/195/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: Map537/events/7/pages/0/195/Dialogue/enemy29
+.........!!
 
+Ah, you're one of those ones?#Ron
 > END STRING
 
 > BEGIN STRING
 …………………………あー、うん。
 
 がんばってね………
-> CONTEXT: Map537/events/7/pages/0/199/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: Map537/events/7/pages/0/199/Dialogue/enemy29
+......Oh... well...
 
+Good luck with that...#Ron
 > END STRING
 
 > BEGIN STRING
 謎の人物は去っていった………
-> CONTEXT: Map537/events/7/pages/0/205/Dialogue < UNTRANSLATED
-
+> CONTEXT: Map537/events/7/pages/0/205/Dialogue
+The mysterious individual left...#Ron
 > END STRING
 
 > BEGIN STRING
 ああああああああああ！！良かったあああ！！
 
 あのままルカくんに手を出してたら、マジで責任問題だよ！
 命令違反で初期化どころの話じゃなくなってたよ！
-> CONTEXT: Map537/events/7/pages/0/214/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: Map537/events/7/pages/0/214/Dialogue/enemy29
+AAAAAAAAAAAH!! THANK GOD!!!
 
+
+
+If I'd actually gotten involved with Luka-kun
+back there, I would've definitely gotten the
+blame! And I'd get far worse than defying orders
+to be formatted!!#RonIffy
```

Comment:
For the latter portion, something like...

> Dealing with Luka directly would have ruined me!
> No way they'd stop at reformatting me for that...

...if you want to shrink this down to 1 text box. If you're willing to drop the extra line, this could be fleshed out more.